src/pages/diary/DiaryDetailPage.jsx

```
@@ -0,0 +1,312 @@
+    } catch (err) {
+      console.error("[DiaryDetailPage] drawingData parse failed", err);
+      return [];
+    }
+  }, [diary?.drawingData]);
+
+  useEffect(() => {
+    const canvas = canvasRef.current;
+    if (!canvas) return;
+    const ctx = canvas.getContext("2d");
+    ctx.clearRect(0, 0, CANVAS_W, CANVAS_H);
+    drawingStrokes.forEach((stroke) => drawStrokeOnCtx(ctx, stroke));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)